tests/test_github.py

```diff
@@ -1913,6 +2005,9 @@ def test_fetch_from_empty_archive(self):
 class TestGitHubClient(unittest.TestCase):
     """GitHub API client tests"""
 
+    def setUp(self):
```

Comment:
I guess this can be removed @Polaris000 ?

Reply:
Done!